virtwho/virt/virt.py

```diff
@@ -316,9 +315,50 @@ def hash(self):
         return hashlib.sha256(json.dumps(self.serializedAssociation, sort_keys=True).encode('utf-8')).hexdigest()
 
 
+class StatusReport(AbstractVirtReport):
+    """
+    Status Report from virt backend about ability to connect.
+    """
+
+    def __init__(self, config, state=AbstractVirtReport.STATE_CREATED):
+        super(StatusReport, self).__init__(config, state)
+        self._source_message = ''
+        self._destination_message = ''
+        self._destination = ''
+
+    @property
+    def data(self):
+        # make the components into dictionary
+        data = {
+            "source": {
+                "server": self.config['server'] if 'server' in self.config else 'none',
```

Comment:
I wonder why do you `'none'` string and not `None`? I'm not telling it is wrong. I just would like to know the reason.

Reply:
It was a placeholder when I started. I think it was for display purposes, but I agree that None is proper.